js/components/Signup.jsx

```diff
@@ -15,6 +15,75 @@ var Signup = React.createClass({
            };
   },
 
+  componentDidMount: function() {
+    // Configure to work with localhost or repcoin.net
+    if (document.domain === 'localhost') {
```

Comment:
Check if we're on localhost or heroku app to see whether we should connect to the test app or the production app